src/components/generic/Button/Button.scss

```diff
@@ -0,0 +1,84 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}--button {
+  border-radius: 1000px;
+  cursor: pointer;
+  font-family: $brand-font-regular;
+  line-height: 1.5rem;
+
+  &:disabled {
+    cursor: auto;
+  }
+}
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}--button.button--join {
+  background: var(--wxc-button--join--normal--bg);
+  border: none;
+  color: var(--wxc-button--join--normal--color);
+  font-size: 16px;
+  padding: var(--wxc-button--join--normal--padding);
+
+  &:hover {
+    background: var(--wxc-button--join--hover--bg);
+  }
+
+  &:active {
+    background: var(--wxc-button--join--active--bg);
+  }
+
+  &:disabled {
+    background: var(--wxc-button--join--disabled--bg);
+    color: var(--wxc-button--join--disabled--color);
+  }
+}
+
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}--button.button--default {
+  background: var(--wxc-button--default--normal--bg);
+  border: var(--wxc-button--default--normal--border);
+  color: var(--wxc-button--default--normal--color);
+  font-size: 16px;
+  padding: 12px;
+
+  .icon {
```

Comment:
we don't know if this class is present or not, can we use display: flex with some align-items value in the button itself?